### PR TITLE
refactor: cypress caller

### PR DESCRIPTION
**Pull request by victorggonzalez**

### CODE REVIEW

File: .github/workflows/cypress-caller.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    # Insert required inputs based on repository
+    with:
+      # Test values
+      node-env-test: test
+      graasp-domain-test: localhost
+      graasp-app-id-test: id-1234567890
+      mock-api-test: true
+    # Insert required secrets based on repository with the following format: ${{ secrets.SECRET_NAME }}
+    secrets:
+      api-host-test: ${{ secrets.REACT_APP_API_HOST_TEST }}
```
Comment (pyphilia) [inline .github/workflows/cypress-caller.yml:26]: Do we really need that?

**Review by pyphilia — APPROVED** [review]
Thanks for the PR @victorggonzalez  do you think we could merge it?

**Comment by juancarlosfarah** [comment]
@pyphilia @spaenleh, do we still need this PR?

**Comment by spaenleh** [comment]
This PR was closed because the functionality was included in #68 with some updates.

**Pull request closed without merging**